allensdk/test/brain_observatory/behavior/test_dprime.py

```diff
@@ -54,24 +66,24 @@ def mock_trials_fixture():
         trials[k] = v
     return trials
 
-from collections import defaultdict
 
 @pytest.fixture
 def mock_rolling_dprime_fixture(mock_trials_fixture):
 
     data_dict = defaultdict(list)
-    for ri, row in mock_trials_fixture[['trial_type', 'response', 'change_time']].iterrows():
+    for ri, row in mock_trials_fixture[['trial_type', 'response',
+                                        'change_time']].iterrows():
         assert not pd.isnull(row['change_time'])
-        if row['trial_type'] == 'go' and row['response'] == True:
+        if row['trial_type'] == 'go' and row['response'] is True:
```

Comment:
Do we need `is` here? Shouldn't it just be `row['response']` or can some of these values be NA?

Reply:
Good catch. This was changed because `== True` fails flake8. I fixed this to be better.